grpc.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.description   = 'Send RPCs from Ruby using GRPC'
   s.license       = 'Apache-2.0'
 
-  s.required_ruby_version = '>= 2.3.0'
+  s.required_ruby_version = '>= 2.4.0'
```

Comment:
does this needs to be fixed in the `templates/` directory too?

Reply:
correct, this has been fixed